Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -190,6 +192,15 @@ static bool isCopyDisplayedPixelBufferAvailable()
 }
 #endif
 
+void MediaPlayerPrivateWebM::responseReceived(PlatformMediaResource& resource, const ResourceResponse& response)
+{
+    ALWAYS_LOG(LOGIDENTIFIER);
+    
+    m_origins.add(SecurityOrigin::create(response.url()));
+    m_resolvedOrigin = SecurityOrigin::create(response.url());
+    m_didPassCORSAccessCheck = resource.didPassAccessControlCheck();
```

Comment:
shouldn't this be &= ?

if CORS failed checks earlier, the entire check is false.